dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -68,5 +69,242 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
 
       assert_equal 10, teacher_application.total_score
     end
+
+    test 'autoscore does not override existing scores' do
+      application_hash = build :pd_teacher1819_application_hash
+      application_hash.merge!(
+        {
+          committed: YES,
+          able_to_attend_single: YES,
+          csp_which_grades: ['12'],
+          csp_course_hours_per_year: Pd::Application::ApplicationBase::COMMON_OPTIONS[:course_hours_per_year].first,
+          previous_yearlong_cdo_pd: ['CS Discoveries'],
+          csp_ap_exam: Pd::Application::Teacher1819Application.options[:csp_ap_exam].first,
+          taught_in_past: ['Hour of Code']
+        }
+      )
+
+      application = create :pd_teacher1819_application, course: 'csp', form_data: application_hash.to_json
+      application.update(regional_partner: (create :regional_partner))
```

Comment:
why update rather than inlining this in the create? If we are to use update, use the bang version so we'll catch failures.